util/statvar_dcid_generator.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A utility to generate dcid for statistical variables."""
+
+import copy
+import re
+
+#Constants
+_QUANTITY_REGEX = r"\[(?P<value>-|-?\d+(\.\d+)?) (?P<quantity>[A-Za-z]+)\]"
+_QUANTITY_RANGE_REGEX = (r"\[(?P<lower_limit>-|-?\d+(\.\d+)?) "
+                         r"(?P<upper_limit>-|-?\d+(\.\d+)?) "
+                         r"(?P<quantity>[A-Za-z]+)\]")
+
+# This is a lookup to ensure "Mean" and "Median" are used in the dcid when
+# statType is meanValue or medianValue"
+_stat_type_lookup = {"meanValue": "Mean", "medianValue": "Median"}
+
+# These are the default properties ignored during dcid generation
+_ignore_props = {"unit", "Node", "memberOf", "typeOf"}
+
+
+def capitalize_word(word):
```

Comment:
you may want to strip any preceding namespace for properties, such as 'dcs:' or 'dcid:'

word = word[word.find(":") +1:]